app/src/main/java/com/gh4a/fragment/StarredRepositoryListFragment.java

```diff
@@ -31,14 +31,20 @@
 import java.util.HashMap;
 
 public class StarredRepositoryListFragment extends PagedDataBaseFragment<Repository> {
+    private static final String EXTRA_USER = "user";
+    private static final String EXTRA_SORT_ORDER = "sort_order";
+    private static final String EXTRA_SORT_DIRECTION = "sort_direction";
+    private static final String SORT = "sort";
+    private static final String DIRECTION = "direction";
```

Comment:
sort and direction should go to StarService. See IssueService for an example how to add those fields.